pkg/cmd/backupdriver/cli/install/install.go

```diff
@@ -115,66 +113,41 @@ func NewCommand(f client.Factory) *cobra.Command {
 }
 
 func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
-	var resources *unstructured.UnstructuredList
-
-	// Check vSphere CSI driver version
-	isCSIInstalled, isVersionOk, err := cmd.CheckCSIInstalled(f)
+	kubeClient, err := f.KubeClient()
 	if err != nil {
-		fmt.Println("CSI driver check failed")
-		isCSIInstalled = false
-		isVersionOk = false
-	}
-	if !isCSIInstalled {
-		fmt.Println("Velero Plug-in for vSphere requires vSphere CSI/CNS and vSphere 6.7U3 to function. Please install the vSphere CSI/CNS driver")
-	}
-	if !isVersionOk {
-		fmt.Printf("vSphere CSI driver version is prior to %s. Velero Plug-in for vSphere requires CSI driver version to be %s or above\n", constants.CsiMinVersion, constants.CsiMinVersion)
+		return errors.Wrap(err, "Failed to get kubeClient")
 	}
 
-	// Check velero version
-	veleroVersion, err := cmd.GetVeleroVersion(f, o.Namespace)
-	if err != nil || veleroVersion == "" {
-		fmt.Println("Failed to get velero version.")
-	} else {
-		if cmd.CompareVersion(veleroVersion, constants.VeleroMinVersion) == -1 {
-			fmt.Printf("WARNING: Velero version %s is prior to %s. Velero Plug-in for vSphere requires velero version to be %s or above.\n", veleroVersion, constants.VeleroMinVersion, constants.VeleroMinVersion)
-		}
-	}
+	// Start with a few of prerequisite checks before installing backup-driver
+	fmt.Println("The prerequisite checks for backup-driver started")
 
-	// Check velero vsphere plugin image repo
-	err = o.CheckPluginImageRepo(f)
-	if err != nil {
-		fmt.Printf("Failed to check plugin image repo, error msg: %s. Using default image %s\n", err.Error(), o.Image)
-	} else {
-		fmt.Printf("Using image %s.\n", o.Image)
-	}
+	// Check vSphere CSI driver version
+	_ = cmd.CheckVSphereCSIDriverVersion(kubeClient)
```

Comment:
error returned is being ignored, any reason for that?, same comment applicable else where.

Reply:
I only refactored the code snippet into a util function here without modifying any original logic. Like in this case, in the original case, the error was ignored. We need to revisit it in a separate change but it is out of scope of this pull request.